#Start speeds for ALFRED
linear_start_speed: 0.5
angular_start_speed: 0.25
decrease_speed: 0.01
increase_speed: 0.01

#D-pad controls
left_button: 11
right_button: 12
forward_button: 13
backward_button: 14

#Start
help_button: 7
#Y
linear_increase_button: 3
#A
linear_decrease_button: 0
#X
angular_increase_button: 2
#B
angular_decrease_button: 1

